backend/tracim_backend/views/core_api/schemas.py

```diff
@@ -1433,12 +1449,35 @@ class LiveMessageSchema(marshmallow.Schema):
     )
 
 
+class LiveMessageSchemaPage(marshmallow.Schema):
+    previous_page_token = marshmallow.fields.String()
+    next_page_token = marshmallow.fields.String()
+    has_next = marshmallow.fields.Bool()
+    has_previous = marshmallow.fields.Bool()
+    per_page = marshmallow.fields.Int()
+    items = marshmallow.fields.Nested(LiveMessageSchema, many=True)
+
+
 class GetLiveMessageQuerySchema(marshmallow.Schema):
     """Possible query parameters for the GET messages endpoint."""
 
-    read_status = marshmallow.fields.String(
-        missing=ReadStatus.ALL.value, validator=OneOf(ReadStatus.values())
+    count = marshmallow.fields.Int(
+        example=10,
+        validate=strictly_positive_int_validator,
+        missing=DEFAULT_NB_ITEM_PAGINATION,
+        default=DEFAULT_NB_ITEM_PAGINATION,
+        allow_none=False,
+    )
+    page_token = marshmallow.fields.String(
+        description="token of the page wanted, if not provided get first" "elements",
+        validate=page_token_validator,
     )
+    read_status = StrippedString(missing=ReadStatus.ALL.value, validator=OneOf(ReadStatus.values()))
+    event_types = EventTypeListField()
+
+    @post_load
+    def live_message_query(self, data: typing.Dict[str, typing.Any]) -> object:
```

Comment:
The type should be `LiveMessageQuery` isn't it?

Reply:
Yes, code is not always as strict as it should for post_load code as this add many redundancy between object and  schema related. 

```suggestion
    def live_message_query(self, data: typing.Dict[str, typing.Any]) -> LiveMessageQuery:
```